### PR TITLE
feat([12828]): lint and prettier all repo

### PR DESCRIPTION
**Pull request by olekid**
feat([12828]): auto remove unused import

### CODE REVIEW

File: packages/postmessage-rpc/tsconfig.json [hunk 1]
```diff
@@ -3,6 +3,6 @@
   "compilerOptions": {
     "outDir": "dist"
   },
-  "include": ["src"],
+  "include": ["src", ".turbo"],
```
Comment (chorniy8pavlo) [inline packages/postmessage-rpc/tsconfig.json:6]: Should .turbo be linted?

File: packages/vaults/src/transaction/config/controllers_update.ts [hunk 2]
```diff
@@ -41,7 +41,7 @@ export class ControllersUpdateTransactionRequest implements TransactionRequest {
     }
 }
 
-export class ControllersUpdateTransactionMapper extends TransactionMapperAbstract<TransactionCandid, ControllersUpdateTransaction> {
+export class ControllersUpdateTransactionMapper extends TransactionMapperAbstract<TransactionCandid> {
```
Comment (chorniy8pavlo) [inline packages/vaults/src/transaction/config/controllers_update.ts:44]: @olsemeno can you please review `vault` package changes?

File: packages/vaults/src/transaction/member/member_remove.ts [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import {TransactionType, VaultRole} from "../../enum/enums";
+import {TransactionType} from "../../enum/enums";
```
Comment (chorniy8pavlo) [inline packages/vaults/src/transaction/member/member_remove.ts:1]: I think it should format it to `{ TransactionType }` automatically

File: turbo.json [hunk 4]
```diff
@@ -3,6 +3,7 @@
   "globalDotEnv": [
     ".env"
   ],
+  "globalEnv": ["VITE_MOCKED_SIGNER_PROVIDER_URL"],
```
Comment (chorniy8pavlo) [inline turbo.json:6]: This is not relevant for other packages except `react-dapp`
If we can specify this only for this example package - would be great

File: examples/react-signer/package.json [hunk 5]
```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "dev": "vite",
     "build": "tsc && vite build",
-    "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
+    "lint": "eslint . --fix --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
```
Comment (chorniy8pavlo) [inline examples/react-signer/package.json:9]: Please make sure that this line is present on every package/**/package.json

**Review by chorniy8pavlo — CHANGES_REQUESTED** [review]

File: packages/vaults/src/transaction/member/member_remove.ts (hunk 3 above)
Comment (olekid) [reply packages/vaults/src/transaction/member/member_remove.ts]: yep, fixed

File: packages/postmessage-rpc/tsconfig.json (hunk 1 above)
Comment (olekid) [reply packages/postmessage-rpc/tsconfig.json]: no, missed

Comment (olekid) [reply packages/postmessage-rpc/tsconfig.json]: no, missed here

File: turbo.json (hunk 4 above)
Comment (olekid) [reply turbo.json]: fixed

**Review by dmitriiIdentityLabs — APPROVED** [review]

**Review by chorniy8pavlo — APPROVED** [review]

**Pull request merged**